packages/framework/src/Resources/config/services/commands.yml

```diff
@@ -37,3 +37,7 @@ services:
     Shopsys\MigrationBundle\Command\GenerateMigrationCommand:
         arguments:
             - '%shopsys.vendor_dir%'
+
+    Shopsys\FrameworkBundle\Command\FixExtendedClassesAnnotationsCommand:
```

Comment:
Is it possible to have this command available in dev env only? 
In the production environment it's not doing anything (compiler pass is not executed) and the command says he did something. This could be confusing.

Reply:
I am thinking about either moving the definition to `services_dev.yml` even though it is a command, or at least updating the description of the command with information that it works in `DEV` environment only

Reply:
in 142df2a36, I have moved the definition to `services_dev.yml` so it is not available in any other environment than `DEV` - it should also fix the failing Travis build as a consequence :slightly_smiling_face: